test/type_test.ts

```diff
@@ -96,6 +96,47 @@ external String Function(String) get x;
 @JS()
 external set x(String Function(String) v);`);
   });
+
+  describe('TypeScript utility types and other mapped types', () => {
+    it('emits X in place of Partial<X> since all Dart types are currently nullable', () => {
+      expectTranslate('interface X { a: number; } declare const x: Partial<X>;')
+          .to.equal(`@anonymous
+@JS()
+abstract class X {
+  external num get a;
+  external set a(num v);
+  external factory X({num a});
+}
+
+@JS()
+external X/*=Partial<X>*/ get x;`);
```

Comment:
remove the = from these comments and other commented types as that is just a remanent of legacy dart syntax for defining types.